src/utils/fetch-config.ts

```diff
@@ -68,6 +69,9 @@ export async function fetchConfig (): Promise<void> {
   window['ldClientId'] = ldClientId
   console.info('Set Launch Darkly Client ID.')
 
+  // initialize Launch Darkly
+  await initLDClient()
+
```

Comment:
I'd prefer to see this init visible in `main.ts` (same as in Create UI).

Reply:
The validation piece is right after this line of code and we need the ldClientId to be set for the `initLDClient()`

Should I move the feature flag check elsewhere?

Reply:
No, I had in mind the FF check here, but I forgot that you need to load the config before init (catch-22).

What do you think about moving this FF check to main.ts and wrapping it around the app start (same as Create UI)? You could then log a message if the flag is false (and/or throw an error that the existing catch logic would handle).

Reply:
Sounds good, will do that